server.sh

```diff
@@ -8,7 +8,7 @@ while true; do
   CURRENT_SONG=$(redis-cli -h "$REDIS_HOST" RPOP current_song)
   if [ "$CURRENT_SONG" != "" ]; then
     youtube-dl --extract-audio --audio-format best --output "current_song.%(ext)s" "$CURRENT_SONG"
-    omxplayer -o both current_song.*
+    omxplayer -o local current_song.*
```

Comment:
define a variable inside .env, inside .env source a .env.local if present (to override defaults)